infra/notifications/src/emailSend.js

```diff
@@ -19,10 +18,140 @@ if (!process.env.SENDGRID_FROM_EMAIL) {
   logger.warn('Warning: SENDGRID_FROM_EMAIL env var is not set')
 }
 
+const {
+  getMessageFingerprint,
+  isNotificationDupe,
+  logNotificationSent
+} = require('./dupeTools')
+
+//
+// Email notifications for Messages
+//
+async function messageEmailSend(receivers, sender, config) {
+  if (!receivers) throw 'receivers not defined'
+  if (!sender) throw 'sender not defined'
+
+  // Load email template
+  const templateDir = `${__dirname}/../templates`
+
+  // Standard template for HTML emails
+  const emailTemplateHtml = _.template(
+    fs.readFileSync(`${templateDir}/emailTemplate.html`).toString()
+  )
+  // Standard template for text emails
+  const emailTemplateTxt = _.template(
+    fs.readFileSync(`${templateDir}/emailTemplate.txt`).toString()
+  )
+
+  const messageSender = Identity.findOne({
+    raw: true,
+    where: {
+      ethAddress: sender
+    }
+  })
+  const messageReceivers = Identity.findAll({
+    raw: true,
+    where: {
+      ethAddress: {
+        [Op.or]: receivers
+      }
+    }
+  })
+  Promise.all([messageSender, messageReceivers]).then(
+    ([senderIdentity, receiversIdentities]) => {
+      receiversIdentities.forEach(async s => {
+        try {
+          const message = messageTemplates.message['email']['messageReceived']
+
+          if (!s.email && config.overrideEmail) {
+            if (config.verbose) {
+              logger.info(`${s.ethAddress} has no email address. Skipping.`)
+            }
+          } else {
+            // Construct best human readable version of sender name
+            const senderName =
+              senderIdentity.firstName || senderIdentity.lastName
+                ? `${senderIdentity.firstName ||
+                    ''} ${senderIdentity.lastName || ''} (${sender})`
+                : sender
+            const templateVars = {
+              config: config,
+              sender: sender,
+              senderName: senderName
+            }
+            const email = {
+              to: config.overrideEmail || s.email,
+              from: config.fromEmail,
+              subject: message.subject(templateVars),
+              text: emailTemplateTxt({
+                message: message.text(templateVars)
+              }),
+              html: emailTemplateHtml({
+                message: message.html(templateVars)
+              }),
+              asm: {
+                groupId: config.asmGroupId
+              }
+            }
+
+            if (config.verbose) {
```

Comment:
As opposed to wrapping those debug statements with config.verbose, how about using logger.debug statement that can be enable via setting env var LOG_LEVEL to DEBUG ?

Reply:
Good point. The whole `--verbose` thing started before @tomlinton showed me the light of our logger over console.log. Agreed that we should ditch verbose and just rely on `LOG_LEVEL`s.